src/mongocxx/test/database.cpp

```diff
@@ -42,6 +42,12 @@ using bsoncxx::builder::basic::make_document;
 
 bool check_for_collections(cursor cursor, std::set<std::string> expected_colls) {
     for (auto&& coll : cursor) {
+        // Skip system collections
```

Comment:
Comment says what you are doing, but not why.

Reply:
Fixed in a separate commit which I'll squash before merge.